<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7QUSM76" name="com.training.sanity.tests.LoginExcelTest" tests="1" failures="1" timestamp="19 May 2020 16:51:40 GMT" time="0.000" errors="0">
  <testcase name="loginDBTest" time="0.000" classname="com.training.sanity.tests.LoginExcelTest">
    <failure type="org.testng.TestNGException" message="
Method public void com.training.sanity.tests.LoginExcelTest.loginDBTest(java.lang.String,java.lang.String) requires a @DataProvider named : excel-inputs1 in class com.training.dataproviders.LoginDataProviders">
      <![CDATA[org.testng.TestNGException: 
Method public void com.training.sanity.tests.LoginExcelTest.loginDBTest(java.lang.String,java.lang.String) requires a @DataProvider named : excel-inputs1 in class com.training.dataproviders.LoginDataProviders
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:265)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:414)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginDBTest -->
</testsuite> <!-- com.training.sanity.tests.LoginExcelTest -->
